lib/bitreserve/operations/all.rb

```diff
@@ -0,0 +1,26 @@
+module Bitreserve
+  module Operations
+    module All
+      module ClassMethods
+        def all
+          data = Request.perform(:get, Kernel.const_get("Bitreserve::Endpoints::#{name.split('::').last.upcase}"))
```

Comment:
can't you du `Bitreserve::Endpoints.const_get(name.split('::').last.upcase)` instead?